redis/protocol_parser.rl

```diff
@@ -81,7 +81,7 @@ blob := any+ >start_blob $advance_blob;
 command := any+ >start_command $advance_command;
 arg = '$' u32 crlf ${ _arg_size = _u32;};
 
-main := (args_count (arg @{fcall command; } crlf) (arg @{fcall blob; } crlf)+) ${_req._state = request_state::ok;};
+main := (args_count (arg @{fcall command; } crlf) (arg @{fcall blob; } crlf)+) ${_req._state = request_state::ok;} >eof{_req._state = request_state::eof;};
```

Comment:
If it's not too hard, would be nice to have test case for this, not just a commit-log comment.

Reply:
Currently we test by redis module, this kind of problem can only be found by sending raw string & checking original response.

I will add a series of my tests to redis-test/, thanks.